templates/index_page.mustache

```diff
@@ -0,0 +1,125 @@
+{{!
+@template tool_wsformat/mytemplate
+
+Admin search input for page header.
+
+Example context (json):
+{
+"action": "http://moodle.local/admin/search.php",
+"query": "themedesigner"
+}
+}}
+{{! Get string from template }}
+{{^items_selected}}
+{{! <h2>Please select a webservice</h2> }}
+{{/items_selected}}
+{{#items_selected}}
+<div
+    style="width: 100%; display: flex;flex-direction: column; margin-bottom: 15rem; margin-top: 2rem; padding-bottom: 2rem; border-top: 1px solid #f2f3f5;">
+    <div style="padding-top: 2rem; display: flex;flex-direction: row;">
+        <div style="
+        display: flex;
```

Comment:
I believe this was already mentioned, but this can be pulled out to a css/scss file in the plugin root, which will make the template a fair bit more readable